internal/idp/keycloak/types.go

```
@@ -216,72 +216,29 @@ type keycloakAuthorizationResource struct {
-
 // Conversion functions for authorization resources
-
-func toKeycloakAuthorizationResource(resource *AuthorizationResource) *keycloakAuthorizationResource {
-	var scopes []*keycloakScope
-	for _, scopeName := range resource.Scopes {
-		scopes = append(scopes, &keycloakScope{
-			Name: scopeName,
-		})
-	}
-
+func toKeycloakAuthorizationResource(resource *idp.AuthorizationResource) *keycloakAuthorizationResource {
```

golangci-lint (2.12.2)

[error] func toKeycloakAuthorizationResource is unused

(unused)